src/validation/enter.your.details.validation.ts

```diff
@@ -0,0 +1,26 @@
+import { body } from "express-validator";
+import { EnterYourDetailsErrorMessages } from "../utils/error_manifests/enter.your.details.errors";
+import { Address } from "private-api-sdk-node/src/services/presenter-account/types";
+
+const BlankValidator = {
+    premises: EnterYourDetailsErrorMessages.PREMISES_BLANK,
+    addressLine1: EnterYourDetailsErrorMessages.ADDRESS_LINE_1_BLANK,
+    townOrCity: EnterYourDetailsErrorMessages.TOWN_OR_CITY_BLANK,
+    postCode: EnterYourDetailsErrorMessages.POST_CODE_BLANK,
+    country: EnterYourDetailsErrorMessages.COUNTRY_BLANK
+};
+
+export const blank_field_validations = (errors: Address = BlankValidator) => {
```

Comment:
This function should be camel case not snake case.
There is no need to pass field error messages in through the arguments since they are hardcoded.
This function can just be a constant array with the validators in.